mmv1/third_party/terraform/website/docs/r/compute_instance_from_template.html.markdown

```diff
@@ -45,7 +45,7 @@ resource "google_compute_instance_from_template" "tpl" {
   name = "instance-from-template"
   zone = "us-central1-a"
 
-  source_instance_template = google_compute_instance_template.tpl.id
+  source_instance_template = google_compute_instance_template.tpl.self_link_unique
```

Comment:
Do you think if it makes sense that we also document `self_link_unique` is recommended for resource references in `source_instance_template` field description?